### PR TITLE
format timezone in modal, format patient medication, reminder

### PR DESCRIPTION
**Pull request by bgoodwinbc**

### CODE REVIEW

File: screen/reminders_screens/reminder_modal.js [hunk 1]
```diff
@@ -245,7 +243,7 @@ function ReminderModal(props) {
         <View style={styles.formRow}>
           <DateInput
             field='recur_to'
-            label='End Date'
+            label='End Date &nbsp; '
```
Comment (ianbrown-bearcode) [inline screen/reminders_screens/reminder_modal.js:246]: Is there a reason why this has two spaces and a non-breaking space, while the start date has just a single space?

File: screen/reminders_screens/reminder_modal.js [hunk 2]
```diff
@@ -255,11 +253,11 @@ function ReminderModal(props) {
           />
         </View>
 
-        <View style={styles.formRow}>
+        <View style={[styles.formRow, {}]}>
```
Comment (ianbrown-bearcode) [inline screen/reminders_screens/reminder_modal.js:256]: This is entirely unnecessary. The [] form allows you to combine multiple hashes if you want to add or override elements of the first hash. You aren't making any changes, so the original hash can be used alone.

File: screen/reminders_screens/reminder_modal.js [hunk 3]
```diff
@@ -67,17 +67,15 @@ function DayOfMonthPicker({ reminder, onDayOfMonthChange }) {
...
-        style={[styles.inputField, { height: 30 }]}
-        activeOutlineColor={themeColor.lightBlue}
-        value={`${dayOfMonth}`}
-        onChangeText={onDayOfMonthChange}
-        mode="outlined"
-        dense='true'
-      />
+      <View style={{width:50}}>
+        <NumberInput
+          style={[styles.inputField, { height: 14, width:50 }]}
```
Comment (ianbrown-bearcode) [inline screen/reminders_screens/reminder_modal.js:72]: This represents a rather significant height change (30 to 14). As long as it looks good, that's fine, just wanted to be sure that that is what you intended.

File: screen/reminders_screens/reminder_modal.js (hunk 1 above)
Comment (bgoodwinbc) [reply screen/reminders_screens/reminder_modal.js:246]: Yea it's a cheap hack to get the input fields to line up visually on screen, if it were to be lost not a big deal, but looks nicer

Comment (ianbrown-bearcode) [reply screen/reminders_screens/reminder_modal.js:246]: OK.

File: screen/reminders_screens/reminder_modal.js (hunk 3 above)
Comment (bgoodwinbc) [reply screen/reminders_screens/reminder_modal.js:72]: I made some changes to this, including maxLength={2}

File: screen/reminders_screens/reminder_modal.js (hunk 2 above)
Comment (bgoodwinbc) [reply screen/reminders_screens/reminder_modal.js]: removed

**Pull request merged**